src/routes/docs/products/functions/develop/+page.markdoc

```diff
@@ -1406,10 +1385,9 @@ require "appwrite"
 include Appwrite
 
 def main(context)
-    client = Client.new
-        .set_endpoint('https://cloud.appwrite.io/v1')
-        .set_project(ENV['APPWRITE_FUNCTION_PROJECT_ID'])
-        .set_key(ENV['APPWRITE_API_KEY'])
+    client = Appwrite::Client.new
+    # Set API key
+    client.set_key(context.req.headers.get['x-appwrite-key']) # API key is automatically provided by Appwrite
```

Comment:
we do `context.req.headers['x-appwrite-key']` elsewhere so let's use that syntax consistently.

```suggestion
    client.set_key(context.req.headers['x-appwrite-key']) # API key is automatically provided by Appwrite
```